include/ibamr/IBFEMethod.h

```diff
@@ -685,6 +685,12 @@ class IBFEMethod : public IBStrategy
      */
     void forwardEulerStep(double current_time, double new_time) override;
 
+    /*!
+     * Advance the positions of the Lagrangian structure using the (explicit)
+     * backward Euler method.
+     */
+    void backwardEulerStep(double current_time, double new_time) override;
```

Comment:
Quick question - why is this implemented but not mentioned in the changelog?

Reply:
Because it is not really backward Euler.

Reply:
Because it is not really backward Euler.

Reply:
It is not a real implementation of backward Euler --- it is basically still forward Euler, but a little more stable.